README.md

```diff
@@ -16,7 +16,7 @@
 
 To use the Swift 3 version, add this to your Podfile (until 2.2 or higher is released):
 ```ruby
-pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git'
+    pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git', :branch => 'swift3'
```

Comment:
my .2$: a working example is always better to get new users than having to figure out these backs & forths and many people are on Xcode 8.0.